build/config/mac/BUILD.gn

```diff
@@ -16,6 +16,7 @@ config("mac_dynamic_flags") {
 
     # Path for loading shared libraries for unbundled binaries.
     "-Wl,-rpath,@loader_path/.",
+    "-Wl,-rpath,/usr/local/lib/.",
```

Comment:
Are we ok with this applying to all Flutter.framework builds for macOS? CAn/should we apply this only in tests?

Reply:
/usr/lib was already in there. So I figured this would be fine too. Your call.

Reply:
This is fine